views/political_information.py

```diff
@@ -1,10 +1,43 @@
 from rest_framework import generics, permissions
 from rest_framework.exceptions import NotFound
-
+from rest_framework.response import Response
+from rest_framework import status
```

Comment:
This should be the import structure:
```
from external_library (django, restframework) import module

from external_app (open_auth, noticeboard, etc.) import module

from current_app (developer) import module
```
Notice the newline in between the groupings.